src/phpDocumentor/Descriptor/Builder/PhpParser/InterfaceAssembler.php

```diff
@@ -28,11 +28,11 @@ class InterfaceAssembler extends AssemblerAbstract
      *
      * @param Interface_ $data
      *
-     * @return InterfaceDescriptor
+     * @return Interface_
      */
     public function create($data)
     {
-        $interfaceDescriptor = new InterfaceDescriptor();
+        $interfaceDescriptor = new Interface_();
```

Comment:
I will have to rewrite the Assemblers in to factory strategies, So this will be removed in a next pr.